.gitignore

```diff
@@ -41,8 +41,5 @@ local-dev-cluster
 /tmp
 .env
 
-# object files
-*.o
-
```

Comment:
should we check in the `.o` ?

Reply:
As above. I'd prefer not to, but getting `go generate ./...` to run before any `go` commands are used in CI + ensuring that `llvm` etc... are installed was non-trivial based on how CI is set up at the moment.
I'd prefer to get this in as-is, then look at removing the checked in object files.

The cilium docs seem to indicate this is desirable. See:
https://github.com/cilium/ebpf/blob/8079b37aa138b38707461cb8d548edf3359d6bc2/docs/ebpf/guides/portable-ebpf.md?plain=1#L9-L15